web/database/migrations/2022_05_13_040630_deepwell_page_vote.php

```diff
@@ -0,0 +1,58 @@
+<?php
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class DeepwellPageVote extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // This hands ownership of the page vote table to DEEPWELL
+
+        // Drop old tables
+        Schema::drop('page_rate_vote');
+
+        // Create new table
+        DB::statement("
```

Comment:
Only thing I wanna note is stuff in this style does lock the solution in to Postgres. I don't think that's problematic right now, but if down the road there was a determination to use a different database this would have to be refactored.

This could be done with the typical schema statements and a conditional `if (env("DB_CONNECTION") == "pgsql"`  to run the check constraint.